robot_skills/src/robot_skills/arm/gripper.py

```diff
@@ -0,0 +1,117 @@
+import rospy
+import PyKDL as kdl
+
+from actionlib import GoalStatus
+from robot_skills.robot_part import RobotPart
+from tue_manipulation_msgs.msg import GripperCommandGoal, GripperCommandAction
+from tue_msgs.msg import GripperCommand
+
+class GripperState(object):
+    """
+    Specifies a State either OPEN or CLOSE
+    """
+    OPEN = "open"
+    CLOSE = "close"
+
+class Gripper(RobotPart):
+    """
+    TODO: add description
+    """
+    def __init__(self, robot_name, tf_listener):
+        """
+        Todo: add docstring
+        Args:
+            robot_name:
+            tf_listener:
+        """
+        super(Gripper, self).__init__(robot_name=robot_name, tf_listener=tf_listener)
+        self.offset = None
+        # TODO: remove occupied by, an interface should be stateless. Store this information elsewhere.
```

Comment:
Interesting! I agree, but where does it go? Any idea's?

Reply:
Either statemachine userdata, ED or something else. But this is something for another PR